cpp/src/Ice/Proxy.cpp

```diff
@@ -447,7 +447,7 @@ Ice::ObjectPrx::_facet(string newFacet) const
 }
 
 ReferencePtr
-Ice::ObjectPrx::_fixed(ConnectionPtr connection) const
+Ice::ObjectPrx::_fixed(ConnectionPtr connection) const // NOLINT(performance-unnecessary-value-param)
```

Comment:
We can fix this one when we require C++20, there is a `dynamic_pointer_cast` version that takes a rvalue reference.